guides/source/active_model_basics.md

```diff
@@ -506,35 +1262,45 @@ Finished in 0.024899s, 240.9735 runs/s, 1204.8677 assertions/s.
 6 runs, 30 assertions, 0 failures, 0 errors, 0 skips
 ```
 
-An object is not required to implement all APIs in order to work with
-Action Pack. This module only intends to guide in case you want all
-features out of the box.
-
 ### SecurePassword
 
-`ActiveModel::SecurePassword` provides a way to securely store any
-password in an encrypted form. When you include this module, a
-`has_secure_password` class method is provided which defines
-a `password` accessor with certain validations on it by default.
+`ActiveModel::SecurePassword` provides a way to securely store any password in
+an encrypted form. When you include this module, a `has_secure_password` class
+method is provided which defines a `password` accessor with certain validations
+on it by default.
 
 #### Requirements
 
-`ActiveModel::SecurePassword` depends on [`bcrypt`](https://github.com/codahale/bcrypt-ruby 'BCrypt'),
-so include this gem in your `Gemfile` to use `ActiveModel::SecurePassword` correctly.
-In order to make this work, the model must have an accessor named `XXX_digest`.
-Where `XXX` is the attribute name of your desired password.
-The following validations are added automatically:
+`ActiveModel::SecurePassword` depends on
+[`bcrypt`](https://github.com/codahale/bcrypt-ruby 'BCrypt'), so include this
+gem in your `Gemfile` to use `ActiveModel::SecurePassword`.
+
+```ruby
+gem "bcrypt"
+```
 
-1. Password should be present.
-2. Password should be equal to its confirmation (provided `XXX_confirmation` is passed along).
+`ActiveModel::SecurePassword` requires you to have a `password_digest`
+attribute.
+
+The following validations are added automatically:
```

Comment:
Need a line break in order for it to proper generate the numbered list:

```suggestion
The following validations are added automatically:

```

<details><summary>Details</summary>
<p>


_before_
![Screenshot 2024-02-19 at 13 25 01](https://github.com/Ridhwana/rails/assets/26328/5c9865d1-f5ba-4442-96cd-d7146e7731c6)

_after_
![Screenshot 2024-02-19 at 13 25 26](https://github.com/Ridhwana/rails/assets/26328/32eb362e-61e4-4d57-ab1b-293a47197654)

</p>
</details>